spec/models/miq_provision/state_machine_spec.rb

```diff
@@ -95,6 +95,23 @@
         expect(vm.reload.evm_owner).to        eq(user)
       end
 
+      it "sets miq group" do
```

Comment:
Copying from commit comment...

Shouldn't this be "#set_owner_group" and live in "spec/models/mixins/miq_provision_mixin_spec.rb"?